### ISSUE TITLE
Certificate expired: Something is still wrong with the deb distribution:

### ISSUE THREAD

**Issue by memsharded** [conan-io/conan#9714]
Something is still wrong with the deb distribution:

```
root@ci:~# dpkg -i conan-ubuntu-64.deb 
Selecting previously unselected package conan.
(Reading database ... 117219 files and directories currently installed.)
Preparing to unpack conan-ubuntu-64.deb ...
Unpacking conan (1.40.3) ...
Setting up conan (1.40.3) ...

root@ci:~# conan --version
Conan version 1.40.3

root@ci:~# conan search -r=conancenter jsoncpp/*
ERROR: HTTPSConnectionPool(host='center.conan.io', port=443): Max retries exceeded with url: /v1/ping (Caused by SSLError(SSLError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:852)'),))

Unable to connect to conancenter=https://center.conan.io
1. Make sure the remote is reachable or,
2. Disable it by using conan remote disable,
Then try again.
```

I also tried to install manually the certificates:
```
root@ci:~# conan config install https://github.com/conan-io/conanclientcert.git
Trying to clone repo: https://github.com/conan-io/conanclientcert.git
Repo cloned!
Copying file LICENSE to /root/.conan/.
Copying file cacert.pem to /root/.conan/.
root@ci:~# conan search -r=conancenter jsoncpp/*
ERROR: HTTPSConnectionPool(host='center.conan.io', port=443): Max retries exceeded with url: /v1/ping (Caused by SSLError(SSLError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:852)'),))

Unable to connect to conancenter=https://center.conan.io
1. Make sure the remote is reachable or,
2. Disable it by using conan remote disable,
Then try again.
```

Can anybody take a look? Our pipelines are stuck.

_Originally posted by @bbossola in https://github.com/conan-io/conan/issues/9695#issuecomment-932794191_

**Comment by memsharded** [comment]
It seems to only happen in .deb, not a pyinstaller issue (Windows installer seems to work fine)

**Comment by memsharded** [comment]
No idea what could be happening:

- I have checked the code of the ``.deb`` creator. Nothing there that could be causing this
- I have checked that the installed .deb actually contains the updated certificate in its ``/root/.conan/cacert.pem`` file.

I'd need someone a bit more experienced in Linux and .deb packages to have a look at this.

**Comment by czoido** [comment]
We have just released Conan 1.40.4 that should fix the problem with deb packages and certificates, please reopen if the issue persists. 
cc/ @bbossola

**Comment by bbossola** [comment]
Great, everything works, tested!
